### PR TITLE
Touch input stuff

### PR DESCRIPTION
**Pull request by kuwoh**
Added touch input stuff from main branch to arcade branch

### CODE REVIEW

**Comment by peppy** [comment]
can this be done via cherry picking commits instead of using github web site? it would greatly reduce the review overhead.

**Comment by kuwoh** [comment]
Not sure how to do that myself, But I'll try doing that next time.

File: osu!stream/Support/Desktop/GameBaseDesktop.cs [hunk 1]
```diff
@@ -59,14 +59,20 @@ protected override SoundEffectPlayer InitializeSoundEffects()
 
         protected override void InitializeInput()
         {
+
+            //InputManager.AddSource(new InputSourceMouse(Window.Mouse));
+            InputSource source;
+
             try
             {
-                InputManager.AddSource(new InputSourceBaanto());
```
Comment (peppy) [inline osu!stream/Support/Desktop/GameBaseDesktop.cs]: you've removed support for the baanto touch screen, which we still use. raw input should be a fallback.

File: osu!stream/osu!stream_desktop.csproj [hunk 2]
```diff
@@ -250,6 +252,7 @@
     <Reference Include="OpenTK">
       <HintPath>OpenTK.dll</HintPath>
     </Reference>
+    <Reference Include="System.Windows.Forms" />
```
Comment (peppy) [inline osu!stream/osu!stream_desktop.csproj]: not sure we want this.

File: osu!stream/Support/Desktop/GameBaseDesktop.cs (hunk 1 above)
Comment (kuwoh) [reply osu!stream/Support/Desktop/GameBaseDesktop.cs]: Oh, my bad.

**Comment by peppy** [comment]
Closing due to lack of review application. Please re-open if you decide to pick this back up.

**Pull request closed without merging**